### PR TITLE
ui: Make zonal dashboard larger

### PR DESCRIPTION
**Pull request by rhtyd**
The host/alert dashboard panels are on the top side, the dashboard
statistics page gets less room. This increases the panel by 100px for
zone dashboard and decreases the host/alert panels by 100px.

## Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

## Screenshots (if appropriate):

screenshot from 2018-04-10 17-35-32

## How Has This Been Tested?

UI changes + refresh.

## Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/apache/cloudstack/blob/master/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
Testing
- [ ] I have added tests to cover my changes.
- [x] All relevant new and existing integration tests have passed.
- [x] A full integration testsuite with all test that can run on my environment has passed.

@blueorangutan package

### CODE REVIEW

**Comment by rhtyd** [comment]
Thanks @rafaelweingartner @DaanHoogland. Just another unrelated thought - would it be nice to out cloudstack screenshots showing various views in action  on our website?

**Comment by rafaelweingartner** [comment]
I have seen this in different websites lately. Sometimes I find it great, because it can give a pretty good idea of the software; however, sometimes I also find it is useless...

I guess it depends on what we want to show, and how we show it.

File: ui/css/cloudstack3.css [hunk 1]
```diff
@@ -4451,7 +4451,7 @@ textarea {
 /**** Head*/
 .dashboard.admin .dashboard-container.head {
   width: 966px;
-  height: 331px;
+  height: 431px;
```
Comment (nitin-maharana) [inline ui/css/cloudstack3.css]: LGTM.

**Comment by rafaelweingartner** [comment]
I merged this PR. However, there is a problem now to merge forward. @DaanHoogland how are you managing the merge forward of that PR that created the upgrade from 4.10 to 4.11? I think to merge things forward now, we need first to merge your PR in master.

**Comment by DaanHoogland** [comment]
yes, @rafaelweingartner makes sense. #2559 needs to go first. @nitin-maharana had some comments on it, i'll address them this morning.

**Pull request merged**